app/templates/search/macros/filters/date_input.html

```diff
@@ -3,9 +3,12 @@
 {% from "components/date-input/macro.html" import tnaDateInput %}
 
 
-{% macro date_input(date_from, date_to, title) %}
-<div class="tna-aside tna-aside--tight tna-background-tint tna-!--margin-top-s">
+{% macro date_input(date_from, date_to, title, hint=False) %}
+<div class="tna-aside tna-aside--tight tna-background-tint tna-!--margin-top-s etna-date-range">
   <h3 class="tna-heading-m">{{ title }}</h3>
+  {% if hint %}
+    <p class="etna-date-range__hint">{{ hint }}</p>
+  {% endif %}
```

Comment:
Would it work if we passed the `hint` into the `tnaDateInput` macro or does that not appear where you would want it?

Are we trying to output a single hint text to save space? If so, I have some concerns because the hint isn't linked to the input fields - we might have to add some extra markup to link them.